RoboFile.php

```diff
@@ -0,0 +1,372 @@
+    $this->taskExec(static::$TERMINUS_EXE)
+      ->args(
+              'drush',
+              $site_name . '.' . $env,
+              'cr'
+          )
+      ->run();
+    $this->taskExec(static::$TERMINUS_EXE)
+      ->args(
+              'drush',
+              $site_name . '.' . $env,
+              'pm-uninstall',
+              'search',
+          )
+      ->run();
+    $this->waitForWorkflow($site_name);
+    $this->taskExec(static::$TERMINUS_EXE)
+      ->args(
+              'drush',
+              $site_name . '.' . $env,
+              '--',
+              'pm-enable',
+              '--yes',
+              'search_api_pantheon',
+              'search_api_pantheon_admin',
+          )
+      ->run();
+    $this->taskExec(static::$TERMINUS_EXE)
+      ->args(
+              'drush',
+              $site_name . '.' . $env,
+              'cr'
+          )
+      ->run();
+  }
+
+  /**
+   * @param string $site_name
+   * @param string $env
+   */
+  public function testEnableRedis(string $site_name, string $env = 'dev') {
+    $this->taskExec(static::$TERMINUS_EXE)
+      ->args('redis:enable', $site_name)
+      ->run();
+    $this->waitForWorkflow($site_name);
+  }
+
+  /**
+   * @param string $site_name
+   * @param string $env
+   */
+  public function demoLoginBrowser(string $site_name, string $env = 'dev') {
+    exec(
+          't3 drush ' . $site_name . '.' . $env . ' -- uli admin',
+          $finished,
+          $status
+      );
+    $finished = trim(join('', $finished));
+    $this->output()->writeln($finished);
+    $this->_exec('open ' . $finished);
+  }
+
+  /**
+   * @param string $text
+   */
+  protected function say($text, string $step_id = 'narration') {
+    $now = new \DateTime();
+
+    $filename = 'narration-' .
+            $now->diff($this->started)->format('%I-%S-%F') . '-' . $step_id . '.m4a';
+    $this->output->writeln('/Users/Shared/' . $filename);
+    return (new Process([
+      '/usr/bin/say',
+      '--voice=Daniel',
+      "--output-file={$filename}",
+      '--file-format=m4af',
+      $text,
+    ], '/Users/Shared'))
+      ->enableOutput()
+      ->setTty(TRUE);
+  }
+
+  /**
+   * @param string $site_name
+   *
+   * @return mixed|null
+   */
+  protected function siteInfo(string $site_name) {
+    try {
+      exec(static::$TERMINUS_EXE . ' site:info --format=json ' . $site_name, $output, $status);
+      if (!empty($output)) {
+        $result = json_decode(join("", $output), TRUE, 512, JSON_THROW_ON_ERROR);
+        return $result;
+      }
+    }
+    catch (\Exception $e) {
+    }
+    catch (\Throwable $t) {
+    }
+    return NULL;
+  }
+
+  /**
+   * @param string $site_name
+   * @param string $env
+   */
+  public function testEnvSolr(string $site_name, string $env = 'dev') {
+    $site_folder = $this->getSiteFolder($site_name);
+    $pantheon_yml_contents = Yaml::parseFile($site_folder . '/pantheon.yml');
```

Comment:
errored out here..
```
PHP Fatal error:  Uncaught Error: Class "Yaml" not found in /Users/brianweaver/repos/search_api_pantheon/RoboFile.php:365
```

should there be a `use` at the top of the file to include this class?